core/arch/arm/plat-vexpress/platform_config.h

```diff
@@ -213,6 +213,9 @@
 #define GICD_OFFSET		0
 #define GICC_OFFSET		0x10000
 
+/* SDP memory: 31MByte reserved, see bios_qemu_tz_arm */
```

Comment:
We could do as for qemu_virt: need to "remove" the SDP memory area from the memory cells provided to linux kernel through DT. 
- Should OPTEE  core do it during the inits ? 
- On qemu_virt, I use to act on bios_qemu_tz_arm to modify the DT.
- should we rather  modify the initial qemu_virt and qemu_armv8 DT ?